### PR TITLE
Fix all imports (relative & ordering)

### PR DESCRIPTION
**Pull request by luanpotter**
# Description

This is thanks to https://github.com/luanpotter/vscode-dart-import/pull/23 !

## Type of change

This is just a refactoring.

### CODE REVIEW

**Review by erickzanardo — APPROVED** [review]
Conflicts...Conflicts everywhere.

Joking aside, LGTM

File: lib/components/timer_component.dart [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import 'dart:ui';
+
 import './component.dart';
```
Comment (spydon) [inline lib/components/timer_component.dart:3]: Should we have a standard for whether they should have `./` maybe?
Because now there is a mix.

**Review by spydon — APPROVED** [review]
Lgtm, just one comment

File: lib/components/timer_component.dart (hunk 1 above)
Comment (luanpotter) [reply lib/components/timer_component.dart:3]: Good point, I will fix my plugin to unify that and fix in a followup.

Comment (luanpotter) [reply lib/components/timer_component.dart:3]: Fixed here https://github.com/flame-engine/flame/pull/501

**Pull request merged**